dts/bindings/input/touch-controller.yaml

```diff
@@ -0,0 +1,30 @@
+# Common fields for touch controllers
+
+properties:
+  touchscreen-size-x:
```

Comment:
would probably go for width and height here

Reply:
done